test/test_browser.py

```diff
@@ -2086,7 +2086,8 @@ def test_cubegeom_pre_regal(self):
   @requires_graphics_hardware
   @no_swiftshader
   def test_cubegeom_pre_relocatable(self):
-    self.btest('third_party/cubegeom/cubegeom_pre.c', reference='third_party/cubegeom/cubegeom_pre.png', args=['-sLEGACY_GL_EMULATION', '-lGL', '-lSDL', '-sRELOCATABLE'])
+    self.set_setting('RELOCATABLE')
```

Comment:
Maybe add a comment here? Otherwise it seems like something that is tempting to "fix" by moving it into an `-sRELOCATABLE`.